pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -86,6 +100,38 @@ def activation(
 
 
 class ConvBlock(SequentialWithOutChannels):
+    r"""ConvBlock from :cite:`ULVL2016`.
+
+    This block contains a :class:`~torch.nn.ReflectionPad2d` followed by a
+    :class:`~torch.nn.Conv2d` with subsequent normalization depending on
+    ``instance_norm`` and an activation function depending on ``impl_params`` and
+    ``instance_norm``. The activation function is either a :class:`~torch.nn.ReLU` or a
+    :class:`~torch.nn.LeakyReLU` with ``slope=0.01``.
```

Comment:
This is a little too detailed. It basically needs to only state that:

- convolution is followed by normalization and activation
- image is reflection-padded to preserve the size

Furthermore, the type of activation should be put below the `Args` section, since it depends on `instance_norm`.